chain/events/filter/index.go

```diff
@@ -672,8 +739,37 @@ func (ei *EventIndex) prefillFilter(ctx context.Context, f *eventFilter, exclude
 
 	// collected event list is in inverted order since we selected only the most recent events
 	// sort it into height order
-	sort.Slice(ces, func(i, j int) bool { return ces[i].Height < ces[j].Height })
-	f.setCollectedEvents(ces)
+	sort.Slice(ces, func(i, j int) bool { return ces[i].height < ces[j].height })
```

Comment:
I'm not sure why we bother with this at all...

Reply:
I agree. It was already here so didn't want to mess with it.